brave/browser-laptop
# Need new strings to handle overdue payments

Fixes #7078


Test Plan:

- Create a new wallet
- Stop browser
- Set `reconcileStamp` `ledger-state.json` to the current timestamp or more than a day in the past.
- Verify that the text indicates "is due" an "is coverdue" instead of suggesting a future event.
